pipeline.py

```diff
@@ -183,9 +188,4 @@
     else:
         print("Not uploading to Google Drive")
```

Comment:
driveby comment: This won't be very helpful in a log. Please replace with 'Skipped uploading to Google Drive' - to at least make it look intentional rather like a failure occurred.